doc/uml/Class_Diagram

```diff
@@ -0,0 +1,49 @@
+@startuml
+class Data{
+-Idenitifer
+-Content
+}
```

Comment:
Spelling should at least be fixed to "identifier". Looking below at "Tag", perhaps this should be called "name" to be consistent with a "rename()" method?